metrics/src/test/java/tech/pegasys/pantheon/metrics/prometheus/PrometheusMetricsSystemTest.java

```diff
@@ -120,23 +112,14 @@ public void shouldCreateObservationsFromTimerWithLabels() {
     assertThat(metricsSystem.getMetrics())
         .usingElementComparator(IGNORE_VALUES) // We don't know how long it will actually take.
         .containsExactlyInAnyOrder(
-            new Observation(RPC, "request", null, asList("method", "bucket", "0.005")),
-            new Observation(RPC, "request", null, asList("method", "bucket", "0.01")),
-            new Observation(RPC, "request", null, asList("method", "bucket", "0.025")),
-            new Observation(RPC, "request", null, asList("method", "bucket", "0.05")),
-            new Observation(RPC, "request", null, asList("method", "bucket", "0.075")),
-            new Observation(RPC, "request", null, asList("method", "bucket", "0.1")),
-            new Observation(RPC, "request", null, asList("method", "bucket", "0.25")),
-            new Observation(RPC, "request", null, asList("method", "bucket", "0.5")),
-            new Observation(RPC, "request", null, asList("method", "bucket", "0.75")),
-            new Observation(RPC, "request", null, asList("method", "bucket", "1.0")),
-            new Observation(RPC, "request", null, asList("method", "bucket", "2.5")),
-            new Observation(RPC, "request", null, asList("method", "bucket", "5.0")),
-            new Observation(RPC, "request", null, asList("method", "bucket", "7.5")),
-            new Observation(RPC, "request", null, asList("method", "bucket", "10.0")),
-            new Observation(RPC, "request", null, asList("method", "bucket", "+Inf")),
-            new Observation(RPC, "request", null, asList("method", "sum")),
-            new Observation(RPC, "request", null, asList("method", "count")));
+            new Observation(RPC, "request", null, asList("method", "0.2")),
+            new Observation(RPC, "request", null, asList("method", "0.5")),
+            new Observation(RPC, "request", null, asList("method", "0.8")),
+            new Observation(RPC, "request", null, asList("method", "0.95")),
+            new Observation(RPC, "request", null, asList("method", "0.99")),
+            new Observation(RPC, "request", null, asList("method", "1.0")),
+            new Observation(RPC, "request_sum", null, singletonList("method")),
+            new Observation(RPC, "request_count", null, singletonList("method")));
```

Comment:
For histograms we deliberately converted the _sum and _count metrics to labels instead of completely separate metrics. It comes out much nicer in the JSON-RPC that way.  The logic is in convertHistogramSampleNamesToLabels and is probably very similar to what summaries would require.

Reply:
Done.  I left the histogram branch in place in case we ever come back to it.